README.md

```diff
@@ -0,0 +1,25 @@
+
+```
+ _   _______  _____ _____ _____   _    _ _____ _   _ _____ ___________ 
+| | | | ___ \/  ___/  ___|_   _| | |  | |_   _| \ | |_   _|  ___| ___ \
+| | | | |_/ /\ `--.\ `--.  | |   | |  | | | | |  \| | | | | |__ | |_/ /
+| | | |    /  `--. \`--. \ | |   | |/\| | | | | . ` | | | |  __||    / 
+| |_| | |\ \ /\__/ /\__/ /_| |_  \  /\  /_| |_| |\  | | | | |___| |\ \ 
+ \___/\_| \_|\____/\____/ \___/   \/  \/ \___/\_| \_/ \_/ \____/\_| \_|
+                                                                       
+                                                                       
+ _____ _____  _   _ _____  _____ _                                     
+/  ___/  __ \| | | |  _  ||  _  | |                                    
+\ `--.| /  \/| |_| | | | || | | | |                                    
+ `--. \ |    |  _  | | | || | | | |                                    
+/\__/ / \__/\| | | \ \_/ /\ \_/ / |____                                
+\____/ \____/\_| |_/\___/  \___/\_____/                                
+
+```
+
+Whoop whoop whoes accited to do computer
```

Comment:
Oh! I think there's a typo here. I think this fixes it. 

```suggestion
Whoop whoop who's excited to do computer??!!?!??
```

Reply:
Ya those look great!